packages/os/load-kernel-modules.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Load additional kernel modules
+RefuseManualStart=true
+RefuseManualStop=true
+After=preconfigured.target link-kernel-modules.service
+Wants=preconfigured.target link-kernel-modules.service
```

Comment:
Same note about dependency ordering - let's keep all of this happening in `preconfigured.target`.